setup.py

```diff
@@ -21,7 +21,10 @@
         "attrs",
         "boltons",
         "pyyaml",
+        "pytest",
```

Comment:
I believe pytest should not be added to setup.py, because it's not required for _use_, only _development_. I'm somewhat less than 100% confident in this, please correct me if I'm mistaken.

Same goes for vcrpy below (if I'm correct here)